ci/travis/ci.sh

```diff
@@ -157,7 +159,10 @@ test_python() {
       -python/ray/tests:test_stress_sharded  # timeout
       -python/ray/tests:test_webui
     )
-    bazel test -k --config=ci --test_timeout=600 --build_tests_only -- "${args[@]}";
+  fi
+  if [ 0 -lt "${#args[@]}" ]; then
```

Comment:
What is this if condition doing? Can you document it?

Reply:
Just checks the array length to be greater than zero.